strider/server.py

```diff
@@ -358,7 +358,7 @@ async def generate_traversal_plan(
 ) -> list[dict]:
     """Generate plans for traversing knowledge providers."""
     query_graph = query.message.query_graph.dict()
-    plans = await generate_plans(query_graph)
+    plans, _ = await generate_plans(query_graph)
```

Comment:
I suspect this is broken. We should write tests for the actual endpoints using FastAPI's TestClient.